2-mandatory/3-financial-times.js

```diff
@@ -23,15 +33,29 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+     let arr=[]
+    for (let article of allArticleTitles){
```

Comment:
You could use `select` function here, passing the condition to filter in your array